src/Mod/Spreadsheet/Gui/Command.cpp

```diff
@@ -450,8 +450,8 @@ CmdSpreadsheetAlignTop::CmdSpreadsheetAlignTop()
 {
     sAppModule = "Spreadsheet";
     sGroup = QT_TR_NOOP("Spreadsheet");
-    sMenuText = QT_TR_NOOP("Align &top");
-    sToolTipText = QT_TR_NOOP("Top-align contents of selected cells");
+    sMenuText = QT_TR_NOOP("Align &Top");

```

Comment:
```suggestion
    sMenuText = QT_TR_NOOP("&Top Align");
```
Pending Discord discussion